### ISSUE TITLE
All searx instances have been bust for weeks (it seems)

### ISSUE THREAD

**Issue by PFCS1976** [searx/searx#1584]
No matter which searx instance I use, they respond to 90% of searches with "error...unexpected crash/captcha required"
This has been going on for 2-3 weeks and so far as I can tell no-one is making any effort to fix it

**Comment by SeymourHiney** [comment]
Have the same problem. It started out only happening some times and now Searx defaults to crash/captcha on every search so it is unusable as a search engine.

**Comment by ghost** [comment]
You can try searching for Searx instances that aren't listed on the project's wiki. They should typically work without any issues.

**Comment by unixfox** [comment]
It's unfortunately because of abuse made by bots (see #543 and #729 for more details). 
I had this same issue with Google on my instance (https://searx.be) but I found some ways to block most of the bots and now it's working fine my instance is not blocked by Google anymore.

**Comment by SeymourHiney** [comment]
Unixfox, would you mind elaborating how to do this....if your methods will work on Windows.

**Comment by unixfox** [comment]
@SeymourHiney You are asking me what I've done to my instance for you to replicate it on your Searx instance hosted on Windows?
If that's the case I pretty much only changed the search URL to a nontraditional URL and blocked every request made to the native/previous/classic search URL (POST request on the root `/` of the domain). By applying this trick I knew I would mostly only receive requests from a real browser.
But I'm working on a plugin for the web server [Caddy](https://caddyserver.com/) (available on Windows too) that will block 99% of the bots with a simple trick, the plugin will be open sourced on GitHub when it's done.

**Comment by SeymourHiney** [comment]
Thanks Unifox for the advice.
As for your plugin, you'll be putting a smile on quite a few faces.

**Comment by unixfox** [comment]
@SeymourHiney hello please checkout my comment here: https://github.com/asciimoo/searx/issues/729#issuecomment-511997511

**Comment by unixfox** [comment]
@return42 duplicate of #729, this issue can be closed.

**Comment by return42** [comment]
@unixfox I will close this issue.  Sorry I didn't had time yet to test you [antibot](https://github.com/asciimoo/searx/issues/729#issuecomment-511997511) solution.  But as far as I can tell, the right solution is to block queries from bots.

If someone else comes here, please take a look at @unifox's [antibot](https://github.com/asciimoo/searx/issues/729#issuecomment-511997511) comment / thanks!

**Comment by SeymourHiney** [comment]
Thank you for having sent this solution. Unfortunately, this is slightly above my skill set/knowledge since I cannot tell if I am suppose to do this in Windows Command Prompt and if it can be used in Firefox (since you only listed Apache and Caddy).

**Comment by unixfox** [comment]
it's meant to be used by searx instance owners not by the visitors. if you don't host a searx instance it won't be useful for you.

**Comment by SeymourHiney** [comment]
Thanks for the turn on.

________________________________
